docs/api-spec.yaml

```
@@ -13,6 +16,8 @@ servers:
 paths:
   /monitoring/metrics:
     get:
+      operationId: get__monitoring_metrics
+      description: "get__monitoring_metrics operation"
       operationId: getMonitoringMetrics
```

YAMLlint (1.38.0)

[error] duplication of key "operationId" in mapping

(key-duplicates)